composite/dns-record-check/src/config.ts

```diff
@@ -0,0 +1,6 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Config } from '@chainlink/types'
+
+export const makeConfig = (): Config => {
```

Comment:
For now I haven't found any use case for config, I just return the default one

Reply:
How does the `dns-record-check` configure the underlying adapter it uses? That underlying adapter requires some configuration, right?

Reply:
Yes that makes sense